mmdeploy/backend/ncnn/wrapper.py

```diff
@@ -102,6 +102,7 @@ def forward(self, inputs: Dict[str,
                 data = input_tensor[batch_id].contiguous()
                 data = data.detach().cpu().numpy()
                 input_mat = ncnn.Mat(data)
+                input_mat = input_mat.clone()
```

Comment:
why do we need to clone here?

Reply:
When I try to test the `hardsigmoid` and `hardswish` ops which support `forward_inplace` in `ncnn`, something wrong happened with the `refcount` of `input` (it seems that an exception occurred to the memory address result a `segmentation fault`). So I clone this here and it works to solve the problem.